docs/how-to/websites-on-ipfs/introduction-to-fleek.md

```diff
@@ -4,4 +4,14 @@ title: Introduction to Fleek
 
 # Introduction to Fleek
 
-Fleek is a service that lets you host a website on IPFS, without the need to install anything on your computer or run an command-line scripts. This guide will cover the basics of Fleek, the benefits of the service, and some limitations it currently has.
+Fleek is a service that lets you host a website on IPFS, without the need to install anything on your computer or run command-line scripts. This guide will cover the basics of Fleek, the benefits of the service, and some limitations it currently has.
+
+IPFS is content addressed, which means when you look up a file, you're asking the network to find a node that stores the content hash. If the content changes, the hash changes. This can be a challenge if you‘re constantly adding/updating content. The process of rehashing, reuploading and [updating domain names](/how-to/websites-on-ipfs/link-a-domain/#dns) can be time consuming.
```

Comment:
```suggestion
IPFS is content addressed, which means when you look up a file, you're asking the network to find a node that stores the content hash. If the content changes, the hash changes. This can be a challenge if you‘re constantly adding or updating content. The process of rehashing, adding, and [updating domain names](/how-to/websites-on-ipfs/link-a-domain/#dns) is time consuming.
```